src/Fields/fieldvector.jl

```diff
@@ -260,27 +260,17 @@ LinearAlgebra.ldiv!(A::LinearAlgebra.LU, x::FieldVector) =
     x .= LinearAlgebra.ldiv!(A, Vector(x))
 
 function LinearAlgebra.norm_sqr(x::FieldVector)
-    Base.sum(value -> LinearAlgebra.norm_sqr(backing_array(value)), _values(x))
+    value_norm_sqrs = UnrolledFunctions.unrolled_map(_values(x)) do value
+        LinearAlgebra.norm_sqr(backing_array(value))
+    end
+    return sum(value_norm_sqrs; init = zero(eltype(x)))
```

Comment:
Is this necessary? `Base.sum(value -> LinearAlgebra.norm_sqr(backing_array(value)), _values(x))` looks much better to me-- there doesn't need to be an intermediately stack-allocated `value_norm_sqrs`